### ISSUE TITLE
[Bug]: Its not discoverable how to minimize share popup

### ISSUE THREAD

**Issue by majkinetor** [mattermost/desktop#3197]
### Checks before filing an issue

- [X] This issue doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/mattermost-server/issues).
- [X] I have checked the [issue tracker](https://github.com/mattermost/desktop/issues) and have not found an issue that matches the one I'm filing.
- [X] This issue is not a troubleshooting question. Troubleshooting questions go here: https://forum.mattermost.com/c/trouble-shoot/16.
- [X] This issue is not a feature request. You can request features and make product suggestions here: https://mattermost.com/suggestions/.
- [X] This issue reproduces on the most recent [stable version](https://github.com/mattermost/desktop/releases/latest), or the most recent [prerelease version](https://github.com/mattermost/desktop/releases) of the Mattermost Desktop App.
- [X] I have read the [contribution guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md).

### Mattermost Desktop Version

5.8.1

### Operating System

Windows 11 x64

### Steps to reproduce

Share a screen, you have this

![image](https://github.com/user-attachments/assets/9409e3f7-7cf9-45e6-ae54-9a349425e9fd)

Way to minimize it is to right click then use Minimize. 

Team of 30 people didn't realize this and are constantnly moving this little window around as it blocks the view (particularly when you share the screen).

### Expected behavior

Add regular minimization button

### Observed behavior

No minimize button

### Log Output

```shell
-
```

**Comment by devinbinnie** [comment]
I'm pretty sure it's expected to be always-on-top, since it's a pretty small window. I don't think it's intended to be minimizable. cc @streamer45 if you want to chime in.

**Comment by streamer45** [comment]
Indeed, it was designed to be always on top to avoid losing track of it since it controls states (e.g., mute/unmute). That said, if it's causing issues, maybe there's something we could improve UX-wise. /cc @matthewbirtch

**Comment by majkinetor** [comment]
Its not small at all once you share a screen.  Then, people are always moving it around as it blocks the view.  The typical solution is to move it to another screen..
